lib/avro_turf/confluent_schema_registry.rb

```diff
@@ -41,14 +41,14 @@ def initialize(
 
   def fetch(id)
     @logger.info "Fetching schema with id #{id}"
-    data = get("/schemas/ids/#{id}")
+    data = get("/schemas/ids/#{id}", idempotent: true)
     data.fetch("schema")
   end
 
   def register(subject, schema)
-    data = post("/subjects/#{subject}/versions", body: { schema: schema.to_s }.to_json)
+    data = post("/subjects/#{subject}/versions", body: { schema: schema.to_s }.to_json, idempotent: true)
```

Comment:
I _don't_ think this can be idempotent, unless the registry does deduplication. If you retry you risk registering the schema _twice_.

Reply:
[47d00dd](https://github.com/dasch/avro_turf/pull/199/commits/47d00dd499b5c62168d3a33731695e749aa32e7c)

I fixed the PR